### ISSUE TITLE
Adding flag -o

### ISSUE THREAD

**Issue by SatoshiNakamotoBitcoins** [JeanLucPons/Kangaroo#36]
Hey Jean-Luc,

It would be nice when you could add flag -o for example (-o output.txt), to create a output file with the found results.

Thank you in advance!

Cheers Satoshi

**Comment by WanderingPhilosopher** [comment]
Are you compiling your own version(s)?

**Comment by SatoshiNakamotoBitcoins** [comment]
@WanderingPhilosopher No...you do??

**Comment by WanderingPhilosopher** [comment]
Yes, I compile my own. I made one compilation that auto saves the private key output. I made it for someone on the channel. 

You can try it out:
https://github.com/WanderingPhilosopher/GPUKangaroo/releases

Run a test on small bit and let me know it works for you and what you think.

**Comment by SatoshiNakamotoBitcoins** [comment]
@WanderingPhilosopher Thank you...works fine...
